oshmem/mca/atomic/ucx/Makefile.am

```diff
@@ -9,7 +9,7 @@
 # $HEADER$
 #
 
-AM_CPPFLAGS = $(atomic_ucx_CPPFLAGS)
+SUBDIRS = ../../spml/ucx
```

Comment:
will it work with out-of-source build (vpath)?
probably need to use $(abs_path) to src dir var

Reply:
actually there are different issues in this way - it failed to re-link modules in installation. I'm working over more "graceful" solution :)

Reply:
alternative solution: https://github.com/open-mpi/ompi/pull/5355